stackslib/src/config/mod.rs

```diff
@@ -4211,6 +4241,22 @@ impl MinerConfigFile {
                 miner_default_config.tenure_cost_limit_per_block_percentage
             };
 
+        let contract_cost_limit_percentage = if let Some(percentage) =
+            self.contract_cost_limit_percentage
+        {
+            if percentage == 100 {
+                None
+            } else if percentage > 0 && percentage < 100 {
```

Comment:
Why not allow 0?  Wouldn't that mean that the miner only processes STX-transfers?